javascript/javascript1/week4/voiceAssistant.js

```diff
@@ -0,0 +1,75 @@
+let nameC = ""
+let todos = []
+function calc(command){
+    console.log(eval(command.substring(8)))
+}
+function getReply(command){
+    let arr = command.split(' ')
+if(command.includes("Hello")) {
+    let givenName = command.split(' ')
+    nameC=givenName[4]
+    console.log(`Nice to meet you ${nameC}`)
+}
+else if(!isNaN(arr[2])){
+    calc(command);
+}
+else {
+    switch(command){
+    case "What is my name":
+        if(nameC!=="") console.log(`${nameC}`)
+        else console.log("name has not yet been mentioned")
+        break;
+    case "Add fishing to my todo":
+        todos.push("fishing")
+        console.log("fishing added to your todo")
+        break;
+    case "Add singing in the shower to my todo":
+        todos.push("singing in the shower")
+        console.log("singing in the shower added to your todo")
+        break;
+    case "Remove fishing from my todo":
+        for(let i=0;i<todos.length;i++){if(todos[i]=="fishing"){todos.splice(i,1)}}
+        console.log("Removed fishing from your todo")
+        break;
```

Comment:
These should be dynamic, meaning:
If I write "Add grocery shopping to my todo", you're code should automatically put grocery shopping in the todo-list. 
You can either remove words from the beginning and end of the string to find the todo task, or if you're feeling extra, you can look into "regex".

Reply:
changed it .. now